cmd/control/cmd/status.go

```diff
@@ -24,7 +24,9 @@ var statusCmd = &cobra.Command{
 		fmt.Println(" * installed: " + version + " (" + commit + ")")
 		resp, err := getDaemonStatus(config.Port)
 		if err != nil {
-			fmt.Println(" * daemon: NOT RUNNING!")
+			fmt.Println(" * RESH-DAEMON IS NOT RUNNING")
```

Comment:
I would like to tell people to create issues and to restart the terminal window when resh-daemon is not running for some reason. (launching a new terminal window launches the resh-daemon so it could fix their issue).
```suggestion
			fmt.Println(" * RESH-DAEMON IS NOT RUNNING")
			fmt.Println(" * Please REPORT this here: https://github.com/curusarn/resh/issues")
			fmt.Println(" * Please RESTART this terminal window")
```